index.js

```diff
@@ -7,6 +7,12 @@ const app = express();
 //import package.json
 const pack = require('./package');
 
+//import categories.json
+const category = require('./src/data/categories.json');
+
+//import products.json
+const product = require('./src/data/products');
```

Comment:
this var name implies that this is only a single product. You can use a more intuitive and descriptive name like 'allProducts'